.circleci/config.yml

```diff
@@ -180,17 +184,24 @@ jobs:
         description: Save docker image as
         type: string
         default: ""
+      platforms:
+        description: Platforms to build for
+        type: string
+        default: "linux/amd64"
+      publish:
+        description: Publish the docker image (multi-platform, all tags)
+        type: boolean
+        default: false
+      release:
+        description: Run the release script
+        type: boolean
+        default: false
     machine:
       image: ubuntu-2204:2022.07.1
       resource_class: medium
       docker_layer_caching: true
     steps:
       - checkout
-      - restore_cache:
-          name: Restore docker build cache
-          key: docker-build-cache
-      - attach_workspace:
```

Comment:
Don't we need this to do the docker load later on?

Reply:
Ah, I intended to remove the `restore_cache` only, since that cache is empty and a remnant of some prior experimentation. But we do need the workspace to attach, which is what is causing the CI failure (yay unified docker build job, catching this early).

Reply:
Unfortunately that didn't appear to fix it

Reply:
Think the `docker save` needed the same docker buildx build-context, let's see if CI works now

Reply:
`docker buildx build` has different defaults than `docker build`; the image needed to be loaded into docker explicitly for the `docker save` command to work. Instead, I'll go for the `tar` output type, to handle multi-platform image persistence correctly